### PR TITLE
🧨 Set up activities logic

### PR DESCRIPTION
**Pull request by p-sad**
Paving the way for #301 
This is a proof of concept for how activities could be represented in the application.

### CODE REVIEW

File: packages/ui/src/common/components/Activities/ActivityContent.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    case 'AppliedOnOpeningEvent':
+      return (
+        <>
+          <Link to="#" onClick={() => showModal({ modal: 'Member', data: { id: activity.membership.id } })}>
+            {activity.membership.handle}
+          </Link>{' '}
+          has applied on the opening{' '}
+          <Link to={`/working-groups/openings/${activity.opening.id}`}>{activity.opening.title}</Link>.
+        </>
+      )
```
Comment (p-sad) [inline packages/ui/src/common/components/Activities/ActivityContent.tsx:24]: Might be worth wrapping these in separate components and returning only the extracted component, good/bad idea?

**Review by jodator — CHANGES_REQUESTED** [review]
@p-sad - just to change the PR state on GH. A short summary of a F2F review:

The activities component has a great potential to grow, so we need to prepare base activity components for an extension without modifying the component's code.

File: packages/ui/src/common/components/Activities/ActivityContent.tsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      return <AppliedOnOpeningContent activity={activity} />
+    case 'BudgetSpendingEvent':
+      return <BudgetSpendingContent activity={activity} />
+    case 'ApplicationWithdrawnEvent':
+      return <ApplicationWithdrawnContent activity={activity} />
+    case 'BudgetSetEvent':
+      return <BudgetSetContent activity={activity} />
+    default:
+      return <div />
+  }
```
Comment (jodator) [inline packages/ui/src/common/components/Activities/ActivityContent.tsx:26]: Not great not terrible ;) Let's refactor that later when we have more activities from other modules.

File: packages/ui/src/working-groups/types/WorkingGroupActivity.ts [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+export interface ApplicationWithdrawnActivity extends BaseActivity {
+  eventType: 'ApplicationWithdrawnEvent'
+  membership: {
+    id: string
+    handle: string
+  }
+  opening: {
+    id: string
+    title: string
+  }
```
Comment (jodator) [inline packages/ui/src/working-groups/types/WorkingGroupActivity.ts:20]: Let's start with `Pick<Member, 'id' | 'handle'>`.

**Review by jodator — CHANGES_REQUESTED** [review]
Last small batch of changes before merge.

**Pull request merged**